### ISSUE TITLE
Convert to Gradle, Maven

### ISSUE THREAD

**Issue by atomixnmc** [nguyenq/VietOCRfx#1]
The main concern is the project are still in form of Netbeant project even if i'm very familiar with Netbean of course... but should we move to use modern structure, with Gradle and binary downloading by chotolatey or apt-get, yum.

**Comment by nguyenq** [comment]
I'm a bit familiar with Maven. Currently, it is Ant based. Sure, we can Mavenize it or else.

**Comment by atomixnmc** [comment]
Yes, Maven is still the most appropriate options. I will try to do some this this direction. Thanks

**Comment by nguyenq** [comment]
Since it would affect the project structure and file history, let me do the Mavenizing piece first. Once I check in the change, then you make a fresh clone and take it further from there.

**Comment by nguyenq** [comment]
I've completed the conversion, which involved massive amount of file relocation. Hope it did not break anything. Please check it out and see if you get any problem running it. If it does not work out for you, we may need to scrap the repository and start from a clean slate. Since the development is still in alpha state, it should not be hard to start over.

**Comment by atomixnmc** [comment]
I've check the new commit. Seems like the dependencies from repo.maven.apache.org can not be reached.

Failed to execute goal on project VietOCRfx: Could not resolve dependencies for project net.sourceforge.vietocr:VietOCRfx:jar:1.0-SNAPSHOT: The following artifacts could not be resolved: com.ice:tar:jar:1.0, uk.co.mmscomputing:sane:jar:1.0.0, net.sourceforge.vietpad:VietKeyInput:jar:1.0.0, net.sourceforge.jacob-project:jacob:jar:1.19.0: Could not find artifact com.ice:tar:jar:1.0 in central (https://repo.maven.apache.org/maven2)

As I check earlier today, this url for example return 404 Not Found
https://repo.maven.apache.org/maven2/net/sourceforge/jacob-project/jacob/1.19.0/jacob-1.19.0.jar

**Comment by nguyenq** [comment]
There are several JARs not existing on Maven repository. I used `maven-install-plugin` to install them from `lib` folder to the local Maven repository.

The stock `pom.xml` created by NetBeans for JavaFX Application is very slow, in that it unpacks all the JARs on every build. The slow speed really hampers the development/debug process. Do you know of a better plugin?
